client/src/js/app/components/navbar.vue

```diff
@@ -10,6 +10,17 @@
     class="tw-hidden tw-z-10 md:tw-flex md:flex-row tw-bg-sidebar-grad-end tw-justify-center tw-my-4 md:tw-mx-auto tw-border tw-border-gray-400 tw-divide-x tw-divide-gray-400"
   >
     <!-- List proposals item -->
+    <keep-alive>
+      <router-link
+        v-if="isStaff"
```

Comment:
Could I have the option of turning this off from the config, so other facilities that don't have this database setup don't have this link